frontend/src/app/util/code_gen.cljs

```diff
@@ -41,6 +41,33 @@
     (when-not (= :none (:stroke-style shape))
       (str/format "%spx %s %s" width style (uc/color->background color)))))
 
+(defn get-size
+  [type values]
+  (let [value (cond
+                (number? values) values
+                (string? values) values
+                (type values) (type values)
+                :else (type (:selrect values)))]
+
+    (if (= :width type)
+      (fmt/format-size :width value values)
+      (fmt/format-size :heigth value values))))
+
+(defn get-layout-orientation
+  [value]
+  (if (= :wrap value)
+    "wrap"
+    "nowrap"))
+
+(defn get-layout-direction
+  [value]
+  (case value
+    :row "row"
```

Comment:
Maybe we can change the keyword now that we still can... once we're in production this will be quite painful.